src/models/sales.ts

```diff
@@ -0,0 +1,97 @@
+import { Model, DataTypes, CreationOptional } from 'sequelize';
+import { connection } from '../database';
+import Product from './products';
+
+class Sale extends Model {
+  declare id: number;
+  declare fullName: string;
+  declare deliveryDate: string;
+  declare direction: string;
+  declare contactNumber: number;
+  declare deliveryTime: string;
+  declare deliveryCost: number;
+  declare tax: number;
+  declare totalWithTax: number;
+  declare amount: number;
+  declare surprise: boolean;
+  declare forwarding: boolean;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+}
```

Comment:
You are not storing here the id of the user that created the sale. That's something that we should take into account, because this way we don't know who did the purchase, only the name, but names can change. let's add an association between the sale model and the user model to store that data